mkdocs.yml

```diff
@@ -73,6 +73,9 @@ pages:
   # API Changes: api/changes.md                 # page-tree = NEED_NEW_PAGE
 - Hooks:
   - Using hooks: hooks.md                        # page-tree = NEED_PAGE_MOVE to /hooks/usage.md
```

Comment:
I like your *"merging them into a single 'Setting up hooks" page"* proposal above. Can we go with that? You asked if there will *"be many other framework-specific stuff"* &mdash; I doubt it. So I think one page is okay. Also, in general, I feel fine about having long pages, as long as the headings are organized nicely, because the theme we're using now creates such a nice nav system to traverse the page sections. 

And I hear you on the downsides of the current scheme for storing all these hooks pages. Regarding *"a single folder for all hooks with a page listing all of them by category"*, I'm open to it. Like you say, the downside to that is that, unless the listing is auto-generated there's a risk of it falling out of sync with the actual pages. But I'm open to restructuring it, certainly. It might not even be too hard to write a small tool which generates a summary page of all hooks. I'm sure @ErichBSchulz would be excited about this. 

@mickadoo for context, the documentation working group (loosely organized, so consider yourself a part of it now, if you like :slightly_smiling_face:) had quite a bit of discussion in #52 about how to store hooks documentation. Honestly, the time it took me to facilitate a rough consensus around this question was a significant fraction of the total time I spent migrating the hooks docs from the wiki. Certainly re-organizing them now would be a comparatively quick task (even with a tool to generate a summary page), but I'm hesitant to open up that can of worms again, as far as getting buy-in from others. Although, some sort of tool to auto-generate a summary page would be quite compelling! I think others would probably be excited about it. So if you're interested in pursuing this change, here's what I'd suggest: open a new issue within this repo where others can weigh in on this proposed change. If there's support for it from others, then I'll follow suit. 

Off the bat I'll say that my personal preferences for hooks storage schemes are as follows (but I don't feel too strongly about this): 

1. (most preferred) All hooks under one menu item (as you propose), with an auto-generated summary page to show categories
1. Current system (hooks under menu items by category) 
1. (least preferred) All hooks under one menu item, with a *manually* generated summary page